prov/efa/src/rxr/rxr_pkt_type_req.c

```diff
@@ -331,14 +331,40 @@ uint32_t *rxr_pkt_req_connid_ptr(struct rxr_pkt_entry *pkt_entry)
 	return NULL;
 }
 
+/**
+ * @brief calculate the exact header size for a given REQ pkt_entry
+ *
+ * @param[in]	pkt_entry		packet entry
+ * @return 	header size of the REQ packet entry
+ */
 size_t rxr_pkt_req_hdr_size_from_pkt_entry(struct rxr_pkt_entry *pkt_entry)
```

Comment:
Probably better to explain why `rxr_pkt_req_hdr_size` cannot be used here.

Reply:
Thanks! I added a comment about the difference between two functions.